tracer/test/test-applications/security/Directory.Build.props

```diff
@@ -1,23 +0,0 @@
-<Project>
```

Comment:
Not anything to do this PR, but testing it caused me to notice that it's still here, but not needed any more, thanks to @andrewlock 's changes.